src/app/core/utils/Input.ts

```diff
@@ -61,7 +60,6 @@ export class Input {
             [[S_KEY], [CONTROL_KEY, COMMAND_KEY, META_KEY]],
             [[D_KEY], [CONTROL_KEY, COMMAND_KEY, META_KEY]],
             [[BACKSPACE_KEY]],
-            [[OPTION_KEY]], // Needed because Alt on Chrome on Windows/Linux causes page to lose focus
```

Comment:
Was this supposed to be removed? I can't see why this would conflict with what this pull request is trying to do.

Reply:
This don't need to be removed. I am not sure why this is removed.